src/css/_agent-table.scss

```diff
@@ -0,0 +1,36 @@
+
+.table-container{
+  width: 100%;
+  @include flexNdirection(column);
```

Comment:
Try to place your mixins at the top of each selector.

Reply:
I thought I had thanks for pointing this out